tox.ini

```diff
@@ -12,7 +12,7 @@ setenv =
 [testenv:docs]
 basepython = python3.5
 deps =
-  sphinx
+  sphinx<1.7.7
```

Comment:
For now I am restricting the version of Sphinx used for the documentation unit test because of a bug in the latest version of Sphinx.